### DISCUSSION TITLE
EsiWACE2 user space Motr release - v1.0

### DISCUSSION THREAD

**Discussion by nezzzu** [Seagate/cortx-motr#1650] [General]
Instructions and rpms for running Motr as a user space program for the EsiWACE2 project. 

<hr /><em>This discussion was created from the release <a href='https://github.com/Seagate/cortx-motr/releases/tag/'>EsiWACE2 user space Motr release - v1.0</a>.</em>

**Comment by hessio** [comment]
Hi @nezzzu this link doesn't seem to work?
